pkg/scalers/azure_eventhub_scaler.go

```diff
@@ -121,6 +122,29 @@ func parseAzureEventHubMetadata(config *ScalerConfig) (*eventHubMetadata, error)
 		meta.eventHubInfo.BlobContainer = val
 	}
 
+	meta.eventHubInfo.Cloud = azure.DefaultCloud
+	if val, ok := config.TriggerMetadata["cloud"]; ok {
+		meta.eventHubInfo.Cloud = val
+	}
+
+	serviceBusEndpointSuffixProvider := func(env az.Environment) (string, error) {
+		return env.ServiceBusEndpointSuffix, nil
```

Comment:
is ServiceBus correct here?

Reply:
If you look at the event hub namespace information in the public cloud, the host name for an Event Hub namespace is <name-of-namespace>.servicebus.windows.net. You can see that the event hub Go SDK uses the service bus endpoint suffix, [here](https://github.com/Azure/azure-event-hubs-go/blob/73b7c0f7b25625f14c9a877ecb73230eba886107/namespace.go#L69).

Reply:
The `namespaceWithAzureEnvironment` is used to create the namespace in the `NewHub` function which we use for getting the event hub client.

Reply:
thanks for explaining this, I though that it was an error ❤️